test/oai/test_client.py

```diff
@@ -105,7 +105,9 @@ def test_chat_completion():
 @pytest.mark.skipif(skip, reason="openai>=1 not installed")
 def test_completion():
     config_list = config_list_from_json(
-        env_or_file=OAI_CONFIG_LIST, file_location=KEY_LOC, filter_dict={"tags": ["gpt-3.5-turbo-instruct"]}
+        env_or_file=OAI_CONFIG_LIST,
+        file_location=KEY_LOC,
+        filter_dict={"model": ["gpt-35-turbo-instruct", "gpt-3.5-turbo-instruct"]},
```

Comment:
Why not use tags here?

Reply:
Do you already have a tag in the OAI_CONFIG_LIST saved in the environment variable?

Reply:
We need documentation about the existing tags in the OAI_CONFIG_LIST saved in the environment variable so that developers know which tags could be used in writing those tests.

Reply:
The original code uses tags and passes test, right?